packages/cloudflare/src/cli/build/utils/extract-project-env-vars.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from "node:fs";
+import * as path from "node:path";
+
+import { parse } from "@dotenvx/dotenvx";
+import type { BuildOptions } from "@opennextjs/aws/build/helper.js";
+
+function readEnvFiles(fileNames: string[], { monorepoRoot, appPath }: BuildOptions) {
```

Comment:
nit: maybe it would be clearer to separate reading the file and merging the values in 2 different functions?

Also a comment to indicate the merge order would we helpful

Reply:
Maybe this can be `readEnvFile` (single file) and you use `.map()` - up to you